masq_lib/src/test_utils/mock_websockets_server.rs

```diff
@@ -230,6 +231,10 @@ impl MockWebSocketsServer {
                 log(do_log, index, "No message waiting");
                 None
             }
+            Err(WebSocketError::IoError(e)) if e.kind() == std::io::ErrorKind::ConnectionReset => {
```

Comment:
This is weird. If the peer has reset the connection, there won't be any more data until another connection is set up--or at least there shouldn't be unless the peer is really insane and the operating system is unusually indulgent. Maybe this is no longer required?

Reply:
happily, it can be removed